filament/src/driver/opengl/ContextManagerWGL.cpp

```diff
@@ -110,11 +111,12 @@ void ContextManagerWGL::terminate() noexcept {
         wglDeleteContext(mContext);
         mContext = NULL;
     }
-    if (mWhdc) {
-        ReleaseHDC(mWhdc);
+    if (mHWnd && mWhdc) {

```

Comment:
I think we want `ReleaseDC` to mirror the call to `GetDC` above:
https://docs.microsoft.com/en-us/windows/desktop/api/winuser/nf-winuser-releasedc